lib/virt_autotest/utils.pm

```diff
@@ -107,6 +108,25 @@ sub trim {
     return $text;
 }
 
+#create a host bridge network interface for sles16 via python script poo#178708
+sub create_host_bridge {
```

Comment:
Now bridge related code is in virtual_network_utils.pm , see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22501/files#diff-509e1257b5aaef8140295ff1e770268d5083c32bf0355f5dca1e828241d50696. Would you like to align code there?

Reply:
Great! Make sense now. Move `create_host_bridge_nm` to` virtual_network_utils.pm`. This allows us to use a library file - `virtual_network_utils.pm`  to hold all bridge related setup code.